wrappers/rust/README.md

```diff
@@ -0,0 +1,38 @@
+### Rust Bindings
+
+For convenience, we also provide rust bindings to the ICICLE library for the following primitives:
+
+- MSM
+- NTT
+    - Forward NTT
+    - Inverse NTT
+- ECNTT
```

Comment:
lines 9-13 aren't currently supported from Rust, maybe best to remove them for now